svc-systems/systems/getresource_test.go

```diff
@@ -454,6 +456,17 @@ func TestGetAllSystems(t *testing.T) {
 			}
 		})
 	}
+	req := systemsproto.GetSystemsRequest{
+		URL: "/redfish/v1/Systems",
+	}
+	GetAllKeysFromTableFunc = func(table string) ([]string, error) {
+		return nil, &errors.Error{}
+	}
+	resp := GetSystemsCollection(&req)
+
+	fmt.Println("Response is **** ", resp)
```

Comment:
Remove this print. please have only valid log messages

Reply:
done